lib/property/property.tests.js

```diff
@@ -331,3 +331,43 @@ function getAdminAccessToken() {
 
   });
 }
+
+function getAdminAccessToken() {
```

Comment:
not really needed to add here since you didn't add anything to property that uses it, but the ticket is fine otherwise and I'll eventually abstract these functions out into test helpers, so I'll approve it as is :)